File: Dice/dice.py
import random

roll_again = "yes"

while roll_again == "yes" or roll_again == "y":
    die = int(input("How many die?\n"))
    print ("Rolling the die...")
    print ("The values are....")
    for num in range(0, die):
        print (random.randint(1, 6))

    roll_again = input("Roll the dices again?\n")
    

File: Dice/dice.js
var roll_again = "yes";

while (roll_again==="yes" || roll_again==="y")
{
    var die = prompt("How many dice?");
    for (var i=0, i<die, i++)
    {
        console.log(Math.random(1, 6));
    }
    roll_again = prompt("Roll the dice again?");
}
